pkg/api/object.go

```diff
@@ -49,3 +52,31 @@ func (h *Handler) RemoveObjectStore(w http.ResponseWriter, r *http.Request) {
 	logger.Debugf("started async deletion of the object store")
 	w.WriteHeader(http.StatusAccepted)
 }
+
+// Gets connection information to the object store in this cluster.
+// GET
+// /objectstore/connectioninfo
+func (h *Handler) GetObjectStoreConnectionInfo(w http.ResponseWriter, r *http.Request) {
+	// TODO: auth is extremely important here because we are returning RGW credentials
+
+	// TODO: support other types of connection info to object store (such as swift)
```

Comment:
do we need to keep work items in the TODO comments? what if we track these instead with issues?